blob/test/aborter.test.ts

```diff
@@ -29,20 +29,20 @@ describe("Aborter", () => {
     } catch (err) {}
   });
 
-  it("Should not abort when calling abort() after request finishes", async () => {
+  it('Should not abort when calling abort() after request finishes', async () => {
```

Comment:
should use `"`

Reply:
😔 Gah sorry. Will fix (also in the rollup config files)

Single quotes are better though 😝